### ISSUE TITLE
is addHotpoint() asynchronous?

### ISSUE THREAD

**Issue by jshenguru** [pattern-x/gemini-viewer-examples#61]
The document.getElementByID() gets nothing right after the viewer.addHotpoint(). Calling document.getElementByID() 10 seconds after the viewer.addHotpoint() will get the DOM object. Is there an event I can wait to run the document.getElementByID() ?

**Comment by Lisp1012** [comment]
> The document.getElementByID() gets nothing right after the viewer.addHotpoint(). Calling document.getElementByID() 10 seconds after the viewer.addHotpoint() will get the DOM object. Is there an event I can wait to run the document.getElementByID() ? Also, it does not get me the HTMLElement, it returns Node.

Yes, you need to get it in the next frame, maybe we'll provide a callback later

**Comment by yanzexuan1** [comment]
@Lisp1012 , How is this issue going on? Is it fixed already?
